custodia/store/sqlite.py

```diff
@@ -110,8 +110,11 @@ def list(self, keyfilter=''):
                     continue
                 if not row[0].startswith(child_prefix):
                     continue
-                value.append(row[0][len(child_prefix):].lstrip('/'))
-
+                result = row[0][len(child_prefix):].lstrip('/')
+                if not self.get(row[0]):
```

Comment:
Looking to the `select` query in the list command:
>        search = "SELECT key FROM %s WHERE key LIKE ?" % self.table
>         key = "%s%%" % (path,)

we only get the `key`, but I need the `value` to distinguish between both entities, so I decided  to do the `get` request in order to see the value for this key, maybe we can change the list request to retrieve all data for the entity instead of just the key value.